src/Tribe/Views/V2/Template/Title.php

```diff
@@ -356,7 +356,7 @@ public function get_posts() {
 				 * setup_template_vars() in src/Tribe/Views/V2/View.php
 				 */
 				$view  = View::make( 'list' );
-				$html  = $view->get_html();
```

Comment:
This was causing the tribe_views_v2_after_setup_loop hook to run and on pages where we had the month view cache it would incorrectly label it as index. When month view is always noindex.

Reply:
Not sure it is required or can be removed completely.

Reply:
I think we can remove it.